services/hdbpp-cluster-reporting/server/models.py

```diff
@@ -48,3 +48,62 @@ def __init__(self, hostname, state, role):
 
     def __repr__(self):
         return "<Server %r" % self.hostname
+
+class Attributes(db.Model):
+    """
```

Comment:
Picky, but can we just rename Attributes Attribute, since it only represents a single attribute. Avoids confusion.

Reply:
Actually it represents all the attributes of specified format (spectrum or scalar) and type (devdouble, …).
So I kept it like this for now.